src/Native/Unix/System.Native/pal_signal.c

```diff
@@ -112,16 +112,14 @@ void* SignalHandlerLoop(void* arg)
         {
             // We're now handling SIGQUIT and SIGINT. Invoke the callback, if we have one.
             CtrlCallback callback = g_ctrlCallback;
-            int rv = callback != NULL ? callback(signalCode == SIGQUIT ? Break : Interrupt) : 0;
-            if (rv == 0) // callback removed or was invoked and didn't handle the signal
+            enum CtrlCode ctrlCode = signalCode == SIGQUIT ? Break : Interrupt;
```

Comment:
I'm not famliiar with this syntax (or I'm spacing). What does the enum keyword do here? Forward declaration?

Reply:
It's like `struct` in C, if you don't add a `typedef` then you need to include the keyword also when declaring a variable.